spacy/displacy/__init__.py

```diff
@@ -120,15 +120,14 @@ def app(environ, start_response):
     return [res]
 
 
-def parse_deps(orig_doc: Doc, options: Dict[str, Any] = {}) -> Dict[str, Any]:
+def parse_deps(doc: Doc, options: Dict[str, Any] = {}) -> Dict[str, Any]:
```

Comment:
Unfortunately, this change would be breaking for users who'd call the function with `parse_deps(orig_doc=...)`, so we'll need to move this particular edit to `v4`. In the meantime, we could update the docstring to be consistent (and edit the typo "do parse" to "to parse")

Reply:
Didn't think of that as a breaking change, good catch! I'll revert.